### PR TITLE
First Commit

### PR DESCRIPTION
**Pull request by cs-sujeet-ahirrao**
First Commit

### CODE REVIEW

File: widget/view.controller.js [hunk 1]
```diff
@@ -68,14 +131,173 @@
...
+            "value": $scope.config.metadata.getSelectedRows[0].new_branch + '-' + $scope.config.metadata.getSelectedRows[0].summary,
+            "type": "primitive"
+          }
+        ]
+      };
+      var queryString = {
+        $limit: ALL_RECORDS_SIZE
+      };
+      return $resource(API.QUERY + 'tasks').save(queryString, queryBody).$promise.then(function (response) {
+        if (response['hydra:member'].length > 0) {
```
Comment (cs-pranali-chavan) [inline widget/view.controller.js]: response['hydra:member'] add check

File: widget/view.controller.js [hunk 2]
```diff
@@ -68,14 +131,173 @@
...
+        }
+        else {
+          _createDummyRecord();
+        }
+      });
+    }
+
+    function _checkDynamicVariables() {
+      $resource(API.WORKFLOW + 'api/dynamic-variable/?limit=1000&name=playbook_wizard_config').get({}).$promise.then(function (response) {
+        if (response['hydra:member'].length > 0) {
```
Comment (cs-pranali-chavan) [inline widget/view.controller.js]: response['hydra:member'] add check

File: widget/view.controller.js [hunk 3]
```diff
@@ -68,14 +131,173 @@
...
+    }
+
+    function _checkTaskRecord() {
+      var queryBody = {
+        "logic": "AND",
+        "filters": [
+          {
+            "field": "name",
+            "operator": "eq",
+            "value": $scope.config.metadata.getSelectedRows[0].new_branch + '-' + $scope.config.metadata.getSelectedRows[0].summary,
```
Comment (cs-pranali-chavan) [inline widget/view.controller.js]: preferably put this in a variable above queryBody and add here.
var _xyz = $scope.config.metadata.getSelectedRows[0].new_branch + '-' + $scope.config.metadata.getSelectedRows[0].summary;

value: _xyz

**Review by cs-pranali-chavan — CHANGES_REQUESTED** [review]
comments added. rest code is appropriate

**Review by cs-pranali-chavan — APPROVED** [review]
changes are appropriate

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**